draft-ietf-webpush-protocol.xml

```diff
@@ -305,7 +305,22 @@
         </t>
       </section>
     </section>
-
+    <section title="Connecting to the Push Service">
+      <t>
+        The push service shares the same default port number (443/TCP) with HTTPS, but MAY
+        also advertise the IANA allocated TCP port 1001 using HTTP alternative services
+        <xref target="I-D.ietf-httpbis-alt-svc"/>:
+      </t>
+      <t>
+        The configuration of this port can potentially be optimized for better timeout
+        characteristics for push notifications than port 443.
+      </t>
+      <t>
+        NAT(s) MUST comply with REQ-5 in <xref target="RFC7231"/> which requires that
+        "the value of the 'established connection idle-timeout' MUST NOT be less than
+        2 hours 4 minutes".

```

Comment:
It's not just about being pragmatic, it's about respecting that network operators will make their own decisions about what they will do.  If they decide that 2 hours is silly, then the rationale we present (a trade-off for low usage) will be more convincing.  There's plenty of evidence to suggest that 2 hours is a little long.

BTW, if you decide to keep the reference, then you need to change it to point to RFC 5382.